Sky Atlas/Sky Atlas.html

```diff
@@ -10957,7 +10957,180 @@ <h1>Sections & Primary Docs</h1>
+            <dfn>A.2.10 - Decision</dfn>
+          </td>
+          <td>Decision</td>
+          <td>Core</td>
+          <td>The documents herein define the decision process for dispute resolution.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.2.10 - Analysis By Core GovOps</dfn>
+          </td>
+          <td>Analysis By Core GovOps</td>
+          <td>Core</td>
+          <td>Core GovOps prepares a confidential analysis of the dispute for the Core Facilitator. This analysis must include a summary of all relevant facts as far as it is possible to ascertain, clear identification of any areas of factual ambiguity or disagreement between the parties, a structured presentation of the arguments in favor of each party, and a recommendation if Core GovOps believes one conclusion is clearly supported by the facts.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.2.10 - Adjudication By Core Facilitator</dfn>
+          </td>
+          <td>Adjudication By Core Facilitator</td>
+          <td>Core</td>
+          <td><p>The Core Facilitator reviews the analysis prepared by Core GovOps along with the arguments presented by the parties and reaches a decision within three (3) working days.</p><p>The Core Facilitator must draft a decision setting out relevant facts and the core reasoning of their decision. In doing so, the Core Facilitator is free to adopt arguments presented by the parties, by Core GovOps, or to advance their own analysis.</p></td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.2.10 - Release Of Decision</dfn>
+          </td>
+          <td>Release Of Decision</td>
+          <td>Core</td>
+          <td>The Core Facilitator communicates their decision to Core GovOps. Core GovOps communicates the decision to all related parties.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.2.10 - Publication Of Decision</dfn>
+          </td>
+          <td>Publication Of Decision</td>
+          <td>Core</td>
+          <td>The Core Facilitator publishes the decision on the Sky Forum and updates the <dfn>A.2.10 - Dispute Resolutions</dfn> Active Data document in the Atlas, redacting any confidential information as needed.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.2.10 - Dispute Resolution Recording</dfn>
+          </td>
+          <td>Dispute Resolution Recording</td>
+          <td>Active Data Controller</td>
+          <td><p>Resolutions of disputes involving Ecosystem Accords are defined as Active Data in A.2.10 - Dispute Resolutions.</p><p>The Active Date is updated as follows:</p>
+            <ul>
+              <li>The Responsible Party is the Core Facilitator</li>
+              <li>The Update Process must follow the protocol for "Direct Edit"</li>
+            </ul>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.2.10 - A2 - Ecosystem Accords - List Of Active Ecosystem Accords</dfn>
```

Comment:
```suggestion
            <dfn>A.2.10 - A2 - Ecosystem Accords - Active Ecosystem Accords</dfn>
```

This suggestion should also be applied to line 11135 but I wasn’t able to leave a suggestion for that line.